File: Service_eHospital_DongNai_A_Dictionary/Implement/DM_LoaiBenhAnService.cs
namespace Service_eHospital_DongNai_A_Dictionary.Implement
{
    public class DM_LoaiBenhAnService : BaseService<DM_LoaiBenhAn, IDM_LoaiBenhAnRepository>
    , IDM_LoaiBenhAnService
    {
    private readonly IDM_LoaiBenhAnRepository _DM_LoaiBenhAnRepository;
    public DM_LoaiBenhAnService(IDM_LoaiBenhAnRepository DM_LoaiBenhAnRepository) : base(DM_LoaiBenhAnRepository)
    {
    _DM_LoaiBenhAnRepository = DM_LoaiBenhAnRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_Dictionary/DM_LoaiBenhAn.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DM_LoaiBenhAn } from './DM_LoaiBenhAn.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DM_LoaiBenhAnService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'LoaiBenhAn_Id', 'NhomLoaiBenhAn_Id', 'MaLoaiBenhAn', 'MaLoaiBenhAn_Map', 'TenLoaiBenhAn', 'TenLoaiBenhAn_En', 'TenLoaiBenhAn_Ru', 'Cap', 'CapTren_Id', 'TamNgung', 'TenKhongDau', 'Property1', 'Property2', 'Property3', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: DM_LoaiBenhAn[] | undefined;
    ListFilter: DM_LoaiBenhAn[] | undefined;
    FormData!: DM_LoaiBenhAn;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DM_LoaiBenhAn";
    }
}

